.github/workflows/check-dependencies.yaml

```diff
@@ -0,0 +1,67 @@
+name: Build and test linkml-runtime package dependencies
```

Comment:
I'm not sure this accurately expresses what this is doing. Maybe change this to something like "Build and test linkml-runtime with latest dependency versions"?

Reply:
sure.